manifests/03-rbac-role-cluster.yaml

```diff
@@ -35,6 +35,17 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - ingresses/status
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - delete
```

Comment:
You should get this for free by adding a consuming user in the `status.componentroute`, can be addressed when the controller is updated to write to status.